specs/phase0/validator.md

```diff
@@ -100,14 +102,42 @@ A validator must initialize many parameters locally before submitting a deposit
 
 Validator public keys are [G1 points](beacon-chain.md#bls-signatures) on the [BLS12-381 curve](https://z.cash/blog/new-snark-curve). A private key, `privkey`, must be securely generated along with the resultant `pubkey`. This `privkey` must be "hot", that is, constantly available to sign data throughout the lifetime of the validator.
 
-#### BLS withdrawal key
+#### Withdrawal credentials
 
-A secondary withdrawal private key, `withdrawal_privkey`, must also be securely generated along with the resultant `withdrawal_pubkey`. This `withdrawal_privkey` does not have to be available for signing during the normal lifetime of a validator and can live in "cold storage".
+The `withdrawal_credentials` ultimately control the deposited ETH once the
+validator has exited and is withdrawable. The 0th byte of this 32-byte field,
+called the "withdrawal prefix" defines the withdrawal credential version
+while the remaining 31 bytes define the version-specific content.
 
-The validator constructs their `withdrawal_credentials` via the following:
+The following withdrawal credentials versions are currently supported. The
+validator must choose and construct a version for the `withdrawal_credentials` field.
 
-* Set `withdrawal_credentials[:1] == BLS_WITHDRAWAL_PREFIX`.
-* Set `withdrawal_credentials[1:] == hash(withdrawal_pubkey)[1:]`.
+##### BLS key credentials
+
+BLS withdrawal credentials are controlled by a secondary withdrawal BLS private key, `bls_withdrawal_privkey`.
+This key is securely generated along with the resultant `bls_withdrawal_pubkey`.
+This `withdrawal_privkey` does not have to be available for signing during
+the normal lifetime of a validator and can live in "cold storage".
+
+The validator constructs `withdrawal_credentials` as the following:
+* Set `withdrawal_credentials[:1] = BLS_WITHDRAWAL_PREFIX`.
+* Set `withdrawal_credentials[1:] = hash(bls_withdrawal_pubkey)[1:]`.
+
+##### Eth1 address credentials
+
+Eth1 address credentials are controlled by an eth1 address. This can be an either an externally owned account or a contract.
+
+The withdrawal to the address specified will be a normal ETH transfer (with no payload other than the validator's ETH)
+triggered by an eth1 transaction that will handle gas price/limit and payment of fees.
```

Comment:
I assume this means that a classic Eth1 account will have to make a call to a precompile that executes the withdrawal? And that Eth1 account will pay for the gas?

Reply:
Yes

The idea is that eth1 is aware of receipts and that a classic eth1 TX can trigger the consumption of a receipt as a send to the destination address. Upon success, the receipt is confirmed as consumed

I'm hesitant to go deep into the details here because they aren't set in stone, but the general "eth1 TX pays for gas to trigger receipt consumption and ETH transfer" is the commitment being made in the PR.  That said, I agree what I've written might not be abundantly clear. *Open to suggestions to clarify*

Reply:
is there any reason to constrain the eth1 account making that call to the withdrawal address? if not, it seems fine to leave this open ended